April 11th 2025 - Matteo Bernardini <ponce@slackbuilds.org>
Fix building with cmake >= 4.x

diff -Naur Clementine-1.4.1.orig/3rdparty/libprojectm/CMakeLists.txt Clementine-1.4.1/3rdparty/libprojectm/CMakeLists.txt
--- Clementine-1.4.1.orig/3rdparty/libprojectm/CMakeLists.txt	2024-10-10 04:42:33.000000000 +0200
+++ Clementine-1.4.1/3rdparty/libprojectm/CMakeLists.txt	2025-04-11 17:57:22.555230000 +0200
@@ -4,11 +4,11 @@
 
 include(CheckCXXCompilerFlag)
 
-cmake_policy(SET CMP0005 OLD)
+cmake_policy(SET CMP0005 NEW)
 
 # This policy was only added in cmake 2.8.4
 if (CMAKE_VERSION VERSION_GREATER "2.8.3")
-  cmake_policy(SET CMP0017 OLD)
+  cmake_policy(SET CMP0017 NEW)
 endif (CMAKE_VERSION VERSION_GREATER "2.8.3")
 
 
diff -Naur Clementine-1.4.1.orig/CMakeLists.txt Clementine-1.4.1/CMakeLists.txt
--- Clementine-1.4.1.orig/CMakeLists.txt	2024-10-10 04:42:33.000000000 +0200
+++ Clementine-1.4.1/CMakeLists.txt	2025-04-11 17:59:11.152230000 +0200
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.0.0)
 project(clementine)
 
-cmake_policy(SET CMP0053 OLD)
+cmake_policy(SET CMP0053 NEW)
 
 include(CheckCXXCompilerFlag)
 include(FindPkgConfig)
